website/data/tools/babel_register/usage.md

```diff
@@ -14,22 +14,22 @@ import "@babel/register";
 
 All subsequent files required by node with the extensions `.es6`, `.es`, `.jsx` and `.js` will be transformed by Babel. The polyfill specified in [polyfill](/docs/usage/polyfill/) is also automatically required.
 
-<blockquote class="alert alert--warning">
+:::caution
   <h4>Not suitable for libraries</h4>
```

Comment:
Can you use move heading to the admonition title? e.g.

```md
:::caution Not suitable for libraries
...
:::
```

Reply:
Just wanted to confirm if we are going to do this for everything? We have a mix of both h3 and h4 headings inside the admonition.

Reply:
Yes I think generally the heading should be the title if we migrate to admonition.

Comparing https://deploy-preview-2731--babel.netlify.app/docs/learn with https://babel.dev/docs/learn.

This PR:
<img width="979" alt="image" src="https://user-images.githubusercontent.com/3607926/221287416-85fce302-2b5f-441f-afdb-5bd8e6b78f59.png">

Current main:
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/3607926/221287577-c9d119fc-edcf-4b63-8015-d1598350e678.png">

The admonition will take up more spaces when both the admonition type and the heading are displayed.

About DocsSearch: currently we configured the agolia bot to crawl `main h3` and `main h4` as lv3 and lv4 indices. I will update the crawler config after this PR gets merged.